core/templates/dev/head/pages/exploration_player/AudioBarDirective.js

```diff
@@ -39,7 +39,7 @@ oppia.directive('audioBar', [
             WindowDimensionsService, AudioTranslationManagerService,
             EVENT_AUTOPLAY_AUDIO, BrowserCheckerService) {
           $scope.audioBarIsExpanded = false;
-          $scope.hasPressedPlayButtonAtLeastOnce = false;
+          $scope.showProgressBar = false;
```

Comment:
`showProgressBar` -> `progressBarIsShown` (start only function name with verbs)

Reply:
but @DubeySandeep has suggested me to do this.

Reply:
@DubeySandeep I think since this a boolean variable, it should not be starting with a verb. Can you please confirm, Thanks!

Reply:
@ankita240796, Thanks for raising this! (Sorry, I have misunderstood the variable as a function and suggested him with that name!)

Reply:
done

Reply:
No issues, thanks for clarifying @DubeySandeep  :smile: